### PR TITLE
Removed "The" prefixes from documentation to make more consise

### LINKED ISSUE #35: [cleanup] Remove "The" prefixes from documentation to make more consise
```
* @param {module:db/CatapultDb} catapultDb The catapult database.
```

=> 
```
* @param {module:db/CatapultDb} catapultDb Catapult database.
```

This will additionally match catapult-server documentation convention.

### PR DESCRIPTION
**Pull request by martinayora**
Fixes #35  Removed "the" prefixes from documentation.

### CODE REVIEW

File: rest/src/plugins/db/MosaicDb.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ class MosaicDb {
 
 	/**
 	 * Retrieves transactions that registered the specified namespaces.
-	 * @param {Array.<module:catapult.utils/uint64~uint64>} namespaceIds The namespaces ids.
+	 * @param {Array.<module:catapult.utils/uint64~uint64>} namespaceIds Namespaces ids.
```
Comment (Jaguar0625) [inline rest/src/plugins/db/MosaicDb.js]: Namespace

Comment (martinayora) [reply rest/src/plugins/db/MosaicDb.js]: done :+1:

**Comment by Vektrat** [comment]
@Jaguar0625 any specific reason/interest why you added bloodyrookie to this review? seems simple enough, just asking to see if we have green light to merge ^^

**Comment by BloodyRookie** [comment]
Looks fine, didn't spot any typos

**Comment by Jaguar0625** [comment]
Can you please also update the new documentation in e2d6eceda4b57e328c37a7e228fe6f2a1a769278.

**Pull request merged**